src/transformers/models/gpt2/tokenization_gpt2_fast.py

```diff
@@ -95,25 +95,23 @@ class GPT2TokenizerFast(PreTrainedTokenizerFast):
     refer to this superclass for more information regarding those methods.
 
     Args:
-        vocab_file (`str`):
+        vocab_file (`str`, *optional*):
```

Comment:
@ArthurZucker 

 I think `vocab_file` is required in order to create a tokenizer, despite we have a default `None` value. Having `*optional*` will create confusion. Not sure what we can do here (for the script)